bin/MadGraph5_aMCatNLO/cards/production/13TeV/VVTo2L2Nu01j_5f_NLO_FXFX/VVTo2L2Nu01j_5f_NLO_FXFX_proc_card.dat

```diff
@@ -0,0 +1,15 @@
+import model loop_sm-ckm_no_b_mass
+#switch to diagonal ckm matrix if needed for speed
+#import model loop_sm-no_b_mass
+
+#include b quark in proton and jet definition for consistent 5 flavour scheme treatment
+define p = p b b~
+define j = j b b~
+
+define ell+ = e+ mu+ ta+
+define ell- = e- mu- ta-
+
+generate p p > ell- vl~ ell+ vl $$ t t~ H [QCD] @0
```

Comment:
according to the madgraph website, particle names are case insensitive (admittedly i checked this now... :) )
http://madgraph.hep.uiuc.edu/EXAMPLES/example.html

If you think for some reason in our context (i.e. we are running some special version of MG?) it may matter, i'd be happy to change it.